Mathlib/Algebra/Opposites.lean

```diff
@@ -343,8 +351,10 @@ theorem op_div [Div α] (a b : α) : op (a / b) = op a / op b :=
   rfl
 #align add_opposite.op_div AddOpposite.op_div
 
+-- porting note: this lemma looked wrong to me -- is it wrong in mathlib3?
```

Comment:
I agree, this is wrong in mathlib3.  We didn't notice this because `/` is the same operation on `αᵃᵒᵖ` and `α` and so it still works as a simp lemma.

Reply:
https://github.com/leanprover-community/mathlib/pull/17955